src/include/duckdb/common/operator/cast_operators.hpp

```diff
@@ -205,4 +205,28 @@ template <> duckdb::dtime_t CastTimestampToTime::Operation(duckdb::timestamp_t i
 template <> duckdb::string_t CastFromTimestamp::Operation(duckdb::timestamp_t input, Vector &result);
 template <> duckdb::timestamp_t CastToTimestamp::Operation(string_t input);
 
+struct CastFromBlob {
+	template <class SRC> static inline string_t Operation(SRC input, Vector &result) {
+		throw duckdb::NotImplementedException("Cast from blob could not be performed!");
+	}
+
+	template <class SRC> static inline void ToHexString(SRC input, SRC &output) {
+		throw duckdb::NotImplementedException("Blob to hexadecimal string could not be performed!");
+	}
+	template <class SRC> static inline string FromHexToBytes(SRC input) {
+		throw duckdb::NotImplementedException("Blob from hex string to bytes could not be performed!");
```

Comment:
Same for this one, this doesn't need to be templated right?

Reply:
There is no special reason for that, I will change.